app/resources/comment_resource.rb

```diff
@@ -9,6 +9,7 @@ class CommentResource < BaseResource
   has_one :parent
   has_many :likes
   has_many :replies
+  has_many :uploads
```

Comment:
hmm, i dont see that option here: jsonapi-resources.com/v0.9/guide/resources.html